### DISCUSSION TITLE
Welcome to oddish Discussions!

### DISCUSSION THREAD

**Discussion by puppylpg** [puppylpg/oddish#51] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by puppylpg** [comment]
interesting, hhh

**Comment by puppylpg** [comment]
So what's the difference between issue and discussion...

**Comment by ccinv** [comment]
大行动来了.学习也忙起来了.Sad

**Comment by puppylpg** [comment]
233，我被buff封号了 🤣

**Reply by xiaoliangxiaoliang** [reply]
你好，大佬，我复制cookie一直出现需要登录的问题，我用的是3.8.1版本的，实在是不知道怎么解决了
